pkg/cmd/sidecar.go

```diff
@@ -44,12 +40,17 @@ func sidecarCommand(c *cli.Context) error {
 		return ErrNotLinux
 	}
 
-	if c.Bool("pprof") {
-		logging.S().Info("starting pprof")
-		go func() {
-			_ = http.ListenAndServe(":6060", nil)
-		}()
-	}
+	startHTTPServer()
 
 	return sidecar.Run(c.String("runner"))
 }
+
+func startHTTPServer() {
+	logging.S().Info("starting http server")
+	go func() {
+		http.HandleFunc("/healthz", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
healthz?

Reply:
At first I thought to check if HTTP on `/healthz` is responding, but then actually just decided to check if TCP server is running on 6060. I will remove it.